packages/vaadin-material-styles/version.js

```diff
@@ -4,18 +4,24 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 
+import { defineCustomElement } from '@vaadin/component-base/src/define.js';
+
 /**
  * Dummy custom element used for collecting
  * development time usage statistics.
  *
  * @private
  */
 class Material extends HTMLElement {
+  static get is() {
+    return 'vaadin-material-styles';
```

Comment:
We probably should do the same for `vaadin-lumo-styles` that also has `version.js`

Reply:
Fixed